cluster/local/integration_tests.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+set -e
+
+# setting up colors
+BLU='\033[0;34m'
+YLW='\033[0;33m'
+GRN='\033[0;32m'
+RED='\033[0;31m'
+NOC='\033[0m' # No Color
+echo_info(){
+    printf "\n${BLU}%s${NOC}" "$1"
+}
+echo_step(){
+    printf "\n${BLU}>>>>>>> %s${NOC}\n" "$1"
+}
+echo_sub_step(){
+    printf "\n${BLU}>>> %s${NOC}\n" "$1"
+}
+
+echo_step_completed(){
+    printf "${GRN} [✔]${NOC}"
+}
+
+echo_success(){
+    printf "\n${GRN}%s${NOC}\n" "$1"
+}
+echo_warn(){
+    printf "\n${YLW}%s${NOC}" "$1"
+}
+echo_error(){
+    printf "\n${RED}%s${NOC}" "$1"
+    exit 1
+}
+
+# ------------------------------
+projectdir="$( cd "$( dirname "${BASH_SOURCE[0]}")"/../.. && pwd )"
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${projectdir} build.vars)
+
+# ------------------------------
+
+SAFEHOSTARCH="${SAFEHOSTARCH:-amd64}"
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-${SAFEHOSTARCH}"
+PACKAGE_IMAGE="crossplane.io/inttests/${PROJECT_NAME}:${VERSION}"
+CONTROLLER_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-controller-${SAFEHOSTARCH}"
+
+version_tag="$(cat ${projectdir}/_output/version)"
+# tag as latest version to load into kind cluster
+PACKAGE_CONTROLLER_IMAGE="${DOCKER_REGISTRY}/${PROJECT_NAME}-controller:${VERSION}"
+K8S_CLUSTER="${K8S_CLUSTER:-${BUILD_REGISTRY}-inttests}"
+
+CROSSPLANE_NAMESPACE="crossplane-system"
+PACKAGE_NAME="provider-kafka"
```

Comment:
Is it a copy-paste leftover? Probably should be parametrized

Reply:
I've updated it. It usually get overridden by the top-level Makefile.

Reply:
Thanks, I've actually seen it exposed in the pipeline run https://github.com/crossplane/provider-template/runs/6307222908?check_suite_focus=true#step:12:79